python/tests/integration/test_streaming_compression_integration.py

```
@@ -19,40 +19,15 @@
 
 import pytest
 
+from .apps import app_source
+
 pytest.importorskip("brotli", reason="brotli package required for httpx auto-decode")
 
 pytestmark = pytest.mark.server_integration
 
 
 def _make_streaming_project(make_server_project):
```

Ruff (0.15.17)

[warning] Missing return type annotation for private function `_make_streaming_project`

(ANN202)